chia/protocols/shared_protocol.py

```diff
@@ -7,7 +7,7 @@
 from chia.util.ints import int16, uint8, uint16
 from chia.util.streamable import Streamable, streamable
 
-protocol_version = "0.0.36"
+protocol_version = "0.0.37"
```

Comment:
Is this still correct?

Reply:
Will need to update this once https://github.com/Chia-Network/chia-blockchain/pull/17791 lands.